pkg/nuclide-external-interfaces/1.0/atom.js

```diff
@@ -735,6 +737,7 @@ declare class atom$TextEditor extends atom$Model {
       screenLine?: boolean,
     },
   ): atom$Point,
+  bufferRowForScreenRow(number): number,
```

Comment:
This is undocumented as well. I don't see why though, it seems like a perfectly reasonable thing to have as part of the public API. I'll let it in since it seems reasonable and there's no obvious non-hacky way to accomplish this using documented APIs.